dist/videojs-errors.js

```diff
@@ -140,14 +138,16 @@
 
     // PLAYER_ERR_NO_SRC
     player.on('play', function() {
-      if (player.currentSrc() === null ||
-          player.currentSrc() === undefined ||
-          player.currentSrc() === '') {
-        player.error({
-          code: -1,
-          type: 'PLAYER_ERR_NO_SRC'
-        });
-      }
+      window.setTimeout(function(){
+        if (player.currentSrc() === null ||
+            player.currentSrc() === undefined ||
+            player.currentSrc() === '') {
+          player.error({
+            code: -1,
+            type: 'PLAYER_ERR_NO_SRC'
+          });
+        }
+      }, 500);
```

Comment:
Looks like play does get fired without a source and with autoplay in Firefox but not Chrome
http://jsbin.com/vubugev/edit?html,console,output